install/default.env

```diff
@@ -21,3 +21,6 @@ MINIO_NOTIFY_MQTT_BROKER_OPENCDMS="tcp://opencdms-broker:1883"
 MINIO_NOTIFY_MQTT_TOPIC_OPENCDMS="fs"
 MINIO_NOTIFY_MQTT_USERNAME_OPENCDMS="opencdms"
 MINIO_NOTIFY_MQTT_PASSWORD_OPENCDMS="insecure-change-me"
+
+# Installation path to sample data
+OPENCDMS_SAMPLE_DATA=/local/app/opencdms/install/opencdms-test-data/data/cdm/
```

Comment:
`OPENCDMS_SAMPLE_DATA=/local/opencdms-test-data/data/cdm/`?

This ultimately comes down to where the clone command is run. It would be good to keep it out of the app directory.

Reply:
In case it's useful - it's possible to run `git clone` from any directory and specify the destination and local repo name, e.g. `git clone https://github.com/opencdms/opencdms /tmp/my-opencdms`